index.html

```diff
@@ -32,12 +32,12 @@ <h4>Relax and Enjoy the world of <span>Men's Fashion</span></h4>
         <div>
             <h1 class='center'><b>HATS</b></h1>
         </div>
-        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint consequatur dicta iure eligendi quibusdam omnis
+        <p>Lorem ipsum dolor sit ame consectetur adipisicing elit. Sint consequatur dicta iure eligendi quibusdam omnis
             vero consequuntur dolores? Blanditiis consectetur quia aliquam, iusto vitae est numquam tempore cupiditate
             tenetur iure!</p>
         <img src="https://cdn.thegentlemansjournal.com/wp-content/uploads/2015/02/hat-TGJ.00.jpg">
```

Comment:
[OPTIONAL]
Hotlinking is not always a good practice. Try downloading the image and displaying from your local repository instead.
Every thing else is good 👍

Reply:
- [ ] Great job spotting 👍🏼 this issue. But as I mentioned before, it will be great to use an explanation for each requested change. How about if you include something like: 
  - "It is considered bad practice to hotlink images or other embedded assets without crediting the author (which many people do). Please, Kindly try downloading the image and displaying it from your local repository instead."
- [ ] Remember that the comment should be more polite. And it should also be a suggestion instead of making it look like an order.